### PR TITLE
Add events data to Security Logs requested from the CLI

### PR DESCRIPTION
**Pull request by pfmaggi**
Print the same information exposed in the `SecurityLogsFragment` when requesting the security logs from the CLI with the command:
```bash
adb shell dumpsys activity service com.afwsamples.testdpc retrieve-security-logs
```

The logic building the StringBuffer has been moved into `Util` to reuse it from both places.

### CODE REVIEW

File: src/main/java/com/afwsamples/testdpc/common/Util.java [hunk 1]
```diff
@@ -354,4 +355,125 @@ public static String constantToString(Class<?> clazz, String prefix, int value)
...
+      case SecurityLog.TAG_CERT_VALIDATION_FAILURE:
+        eventTag = "CERT_VALIDATION_FAILURE";
+        break;
+      default:
+        eventTag = "UNKNOWN(" + eventId + ")";
+    }
+    return eventTag;
+  }
+
+  public static void printData(StringBuilder sb, Object data) {
```
Comment (jscott1989) [inline src/main/java/com/afwsamples/testdpc/common/Util.java:467]: Could we leave this in SecurityLogsFragment and call it statically from there? then it's more obvious that this method is only related to security logs?

Comment (pfmaggi) [reply src/main/java/com/afwsamples/testdpc/common/Util.java]: Sounds good.
Done

File: src/main/java/com/afwsamples/testdpc/common/Util.java [hunk 2]
```diff
@@ -354,4 +355,112 @@ public static String constantToString(Class<?> clazz, String prefix, int value)
   private static String constNameWithoutPrefix(String prefix, Field field) {
     return field.getName().substring(prefix.length());
   }
+
+  public static String getStringEventTagFromId(int eventId) {
```
Comment (jscott1989) [inline src/main/java/com/afwsamples/testdpc/common/Util.java:359]: I think you forgot to remove this? (it can also be statically referenced)

Comment (pfmaggi) [reply src/main/java/com/afwsamples/testdpc/common/Util.java]: Ok, I reverted all the changes to `Util.java` and made the two methods in `SecurityLogsFragment.java` static. I've also cleanup the formatting mess that my editor did in `ShellCommand.java`.
The diff looks a bit cleaner now.

**Review by jscott1989 — APPROVED** [review]

**Pull request merged**